ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -170,6 +170,23 @@ func (r *RayClusterReconciler) Reconcile(ctx context.Context, request ctrl.Reque
 	return ctrl.Result{}, client.IgnoreNotFound(err)
 }
 
+func (r *RayClusterReconciler) DeleteAllPods(ctx context.Context, namespace string, filterLabels client.MatchingLabels) (active int, err error) {
```

Comment:
Do we need to export this function? 

```suggestion
func (r *RayClusterReconciler) deleteAllPods(ctx context.Context, namespace string, filterLabels client.MatchingLabels) (active int, err error) {
```

Reply:
No, we don't.